src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindBreakpointEvidenceSpark.java

```diff
@@ -130,6 +130,8 @@ public boolean requiresReads()
     @Override
     protected void runTool( final JavaSparkContext ctx ) {
 
+        validateParams();
```

Comment:
Does `StructuralVariationDiscoveryPipelineSpark` need to call this too? 

Would it make sense to move this method to the argument collection class itself? Then the tools that use it could call `validate` on it.

Reply:
Yes. That's better. Though each tool should have its own `validate` as well preferably.

Just talked with Engine team, such CLI-arg validation methods must be called explicitly right now, so automatically validating them in the constructor is not supported, yet.